packages/eslint-config-nextjs-ts/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@infinumjs/eslint-config-nextjs-ts",
+  "version": "3.2.0",
+  "description": "Infinum's ESLint NextJS TypeScript rules",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/infinum/js-linters.git"
+  },
+  "author": "Infinum JavaScript Team <hello@infinum.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/infinum/js-linters/issues"
+  },
+  "homepage": "https://github.com/infinum/js-linters#readme",
+  "dependencies": {
+    "@infinumjs/eslint-config-react-ts": "^3.2.0"
```

Comment:
Add `eslint-config-next` as dep

Reply:
In Next.js they have:

Then we recommend either removing these settings if you prefer how these properties have been configured within [eslint-config-next](https://github.com/vercel/next.js/blob/canary/packages/eslint-config-next/index.js) or extending directly from the Next.js ESLint plugin instead:
```
module.exports = {
  extends: [
    //...
    'plugin:@next/next/recommended',
  ],
}
```

So I think we do not need it.